nextcord/activity.py

```diff
@@ -706,6 +711,14 @@ def party_id(self) -> str:
         """:class:`str`: The party ID of the listening party."""
         return self._party.get('id', '')
 
+    @property
+    def elapsed(self) -> datetime.timedelta:
+        """:class:`datetime.timedelta`: Time elapsed since this song.
+
+        .. versionadded:: 2.0
+        """
+        return datetime.datetime.now(datetime.timezone.utc) - self.start
```

Comment:
https://nextcord.readthedocs.io/en/latest/api.html#nextcord.utils.utcnow should be prefered